book/layout.md

```diff
@@ -233,8 +220,7 @@ whether it should have block or inline contents:
 def has_block_children(self):
     for child in self.node.children:
         if isinstance(child, TextNode):
-            if not child.text.isspace():
```

Comment:
Why did these two lines get removed?

Reply:
Thanks to there no longer being whitespace-only TextNodes, after changes in Ch4.